code/modules/reagents/reagents/drugs.dm

```diff
@@ -190,7 +190,7 @@
 /datum/reagent/drug/hyperzine/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
 	if(prob(5))
 		M.emote(pick("twitch", "blink_r", "shiver"))
-	M.add_chemical_effect(CE_SPEEDBOOST, 1)
+	M.add_chemical_effect(CE_SPEEDBOOST, 0.4)
```

Comment:
Don't axe values of game balancing things too much. Lower it to 0.6 and see if it's good in game. With metabolism nerf, you can simple make this chem worseless by nerfing it too hard.

Reply:
Changes made. Let it be known you are making this 0.4 if someone complains about powergamers and hyperzine again